web/src/beta/lib/core/engines/Cesium/Feature/Polyline/index.tsx

```diff
@@ -50,7 +52,18 @@ export default function Polyline({ id, isVisible, property, geometry, layer, fea
     [coordinates ?? []],
     isEqual,
   );
-  const material = useMemo(() => toColor(strokeColor), [strokeColor]);
+
+  const { viewer } = useCesium();
+  const entity = viewer ? findEntity(viewer, layer?.id, feature?.id) : undefined;
```

Comment:
```suggestion
  const entity = useMemo(() => viewer ? findEntity(viewer, layer?.id, feature?.id) : undefined, []);
```
This process should be wrapped by useMemo, because this function runs too heavy process.